Framework/Core/src/WorkflowCustomizationHelpers.cxx

```diff
@@ -56,7 +56,9 @@ std::vector<ConfigParamSpec> WorkflowCustomizationHelpers::requiredWorkflowOptio
 
            // options for AOD writer
            {"aod-writer-json", VariantType::String, "", {"Name of the json configuration file"}},
+           {"aod-writer-resdir", VariantType::String, "", {"Name of the output directory"}},
            {"aod-writer-resfile", VariantType::String, "", {"Default name of the output file"}},
+           {"aod-writer-maxfilesize", VariantType::Float, 0.0f, {"Maximum size of an output file in megabytes"}},
```

Comment:
Do you really want this as a float?

Reply:
For testing it was good to be able to set small values (some kilobytes). That is primarilly why it is a float.

Reply:
Ok with me, then.